### PR TITLE
Hide the db password in stderr

### PR DESCRIPTION
**Pull request by Niveditha-phData**

### CODE REVIEW

File: bin/control.sh [hunk 1]
```diff
@@ -61,7 +61,10 @@ case ${COMPONENT} in
                 FLYWAY_EXECUTABLE="$FLYWAY_DIR/flyway"
                 SCRIPTS_LOCATION="$FLYWAY_DIR/$DATABASE_SCRIPT_DIR"
 
+                echo FLYWAY_LOCATIONS="filesystem:$SCRIPTS_LOCATION" "$FLYWAY_EXECUTABLE" migrate -url="$DB_URL" -user="$DB_USERNAME" -password="" || exit 1
```
Comment (afoerster) [inline bin/control.sh]: This does the job, but there's a risk of changing the command without changing the 'echo'd command.

How about storing the command in a variable? Then you can execute and echo it. When you echo it you will have to set unse the password with `env` `env PASSWORD=hidden echo command`

Comment (Niveditha-phData) [reply bin/control.sh]: If we store the command in variable, that will also be displayed because of `set -x`

File: bin/control.sh [hunk 2]
```diff
@@ -61,7 +61,11 @@ case ${COMPONENT} in
                 FLYWAY_EXECUTABLE="$FLYWAY_DIR/flyway"
                 SCRIPTS_LOCATION="$FLYWAY_DIR/$DATABASE_SCRIPT_DIR"
 
-                env FLYWAY_LOCATIONS="filesystem:$SCRIPTS_LOCATION" "$FLYWAY_EXECUTABLE" migrate -url="$DB_URL" -user="$DB_USERNAME" -password="$DB_PASSWORD" || exit 1
+                # To hide the db password, the flyway command is displayed with all arguments except password and then executed with password in hidden mode
+                FLYWAY_COMMAND="filesystem:$SCRIPTS_LOCATION $FLYWAY_EXECUTABLE migrate -url=$DB_URL -user=$DB_USERNAME -password="
+                set +x
+                env FLYWAY_LOCATIONS=${FLYWAY_COMMAND}$DB_PASSWORD || exit 1
```
Comment (afoerster) [inline bin/control.sh]: You can put `FLYWAY_LOCATIONS=` into the above command on line 65, no reason to set that here

Comment (afoerster) [reply bin/control.sh]: Where do you echo out the command for debugging?

Comment (afoerster) [reply bin/control.sh]: Oh it's done because of set -x

Comment (Niveditha-phData) [reply bin/control.sh]: Line 65 - to store the command without password; as set -x is there, the command is expanded and echoed
Line 67- where the command is executed with password; it is hidden

Comment (afoerster) [reply bin/control.sh]: @Niveditha-phData how about this?
> You can put FLYWAY_LOCATIONS= into the above command on line 65, no reason to set that here

**Review by afoerster — APPROVED** [review]
LGTM

**Pull request merged**